src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1407,12 +1407,6 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
   <data name="HelpLine3.Text" xml:space="preserve">
     <value>Example:  %ReX means red value in hex uppercase two digits format.</value>
   </data>
-  <data name="Help_blue.Text" xml:space="preserve">
```

Comment:
@jaimecbernardo 
You removed both. Is there a third one?

Reply:
There's a third one earlier in the file. :)

Reply:
https://github.com/microsoft/PowerToys/blob/bdbf06bc3f1db3cd836aac83c0c0339a765e2921/src/settings-ui/Settings.UI/Strings/en-us/Resources.resw#L1308